.mailmap

```diff
@@ -1,66 +1,27 @@
-Aron Ahmadia <aron@ahmadia.net>
-Phillip Alderman <phillip.d.alderman@gmail.com>
-James Allen <james@sharelatex.com> <jamesallen0108@gmail.com>
-Piotr Banaszkiewicz <piotr@banaszkiewicz.org>
-Pauline Barmby <pbarmby@uwo.ca>
-Radovan Bast <bast@users.noreply.github.com>
-Olga Botvinnik <olga.botvinnik@gmail.com>
-Andy Boughton <pchemist@gmail.com>
-Erik M. Bray <embray@stsci.edu>
-Amy Brown <arbrown@latte.ca>
-C. Titus Brown <titus@idyll.org>
+Abigail Cabunoc Mayes <abigail.cabunoc@gmail.com>
```

Comment:
Which one? This one and the one below are for a different emails (both appearing in git log, both with "Mayes").